obspy/io/sac/tests/test_core.py

```diff
@@ -84,6 +84,12 @@ def test_read_xy_write_xy_via_obspy(self):
             tempfile = tf.name
             tr.write(tempfile, format='SACXY')
             tr1 = read(tempfile)[0]
+
+        sac = tr.stats.pop('sac', None)
+        sac1 = tr1.stats.pop('sac', None)
+        del sac
+        del sac1
```

Comment:
aha, the holy flake8 :). That explains it, thank you.

What about the following?
```
del tr.stats['sac']
# or more explicitly bypassing flake8
tr.stats.pop('sac')  # NOQA
```